StripeFinancialConnections/StripeFinancialConnections/Source/Native/InstitutionPicker/InstitutionTableViewCell.swift

```diff
@@ -74,14 +74,17 @@ final class InstitutionTableViewCell: UITableViewCell {
 
 extension InstitutionTableViewCell {
 
-    func customize(with institution: FinancialConnectionsInstitution) {
+    func customize(with institution: FinancialConnectionsInstitution, theme: FinancialConnectionsTheme) {
+        self.theme = theme
         institutionIconView.setImageUrl(institution.icon?.default)
 
         institutionCellView.customize(
             iconView: institutionIconView,
             title: institution.name,
             subtitle: AuthFlowHelpers.formatUrlString(institution.url)
         )
+
+        contentView.addAndPinSubview(institutionCellView)
```

Comment:
as mentioned in here:

https://github.com/stripe/stripe-ios/pull/3811/files#r1688386941

this is likely a no-op moving it here, but its "logically" off because technically we should only need to add a view once; if this was a large app, it could have caused perf issues (depending on how iOS handles this - ex. does it treat this as no-op, or does it re-add the view and constraints)

(I only noticed this because its similar to a common performance/coding problem of UITableView/UICollectionView where views are constructed in "customize-type-of-method", which causes these methods to be called on every scroll)

Reply:
I'm not too concerned about the view being added more than once, since that would be caused by developer error, but what I could do is add an `institutionCellView.removeFromSuperview()` before it is added in the `customize` call to prevent this from being an issue.

Reply:
Actually, without this change, this introduced a bug where the view _was_ being added multiple times when scrolling the Institution table view: 

<img src="https://github.com/user-attachments/assets/fe995056-2ba6-4340-961c-ce3f0141e1aa" width=40%>

But this `removeFromSuperview()` change fixes that: 

https://github.com/user-attachments/assets/61740ac0-7a3a-4947-821f-9f1e5e052b9f

Reply:
sweet; these somewhat-minor changes completely shift the code around and it immediately compiles in my head 😅